### PR TITLE
Add events for TWiR 479

### PR DESCRIPTION
**Pull request by mariannegoldin**

### CODE REVIEW

File: draft/2023-01-25-this-week-in-rust.md [hunk 1]
```diff
@@ -180,36 +172,36 @@ Rusty Events between 2023-01-25 - 2023-02-22 🦀
...
     * [**Rust Paris meetup #55**](https://www.meetup.com/rust-paris/events/290100223/)
 * 2023-01-26 | Copenhagen, DE | [Copenhagen Rust Meetup Group](https://www.meetup.com/copenhagen-rust-meetup-group/)
     * [**Rust Hack Night #32**](https://www.meetup.com/copenhagen-rust-meetup-group/events/290037532/)
+* 2023-02-02 | Berlin, DE | [Prenzlauer Berg Software Engineers](https://www.meetup.com/prenzlauer-berg-software-engineers/)
+    * [**PBerg engineers - inaugural meetup; Lukas: Serverless APIs using Rust and Azure functions (Fee)**](https://www.meetup.com/prenzlauer-berg-software-engineers/events/290513923/)
 * 2023-02-02 | Hamburg, DE | [Rust Meetup Hamburg](https://www.meetup.com/rust-meetup-hamburg/)
     * [**Rust Hack & Learn February 2023**](https://www.meetup.com/rust-meetup-hamburg/events/290824576/)
 * 2023-02-02 | Lyon, FR | [Rust Lyon](https://mobilizon.fr/events/b8577678-d072-4d9a-9562-974715f1dfbb)
     * [**Rust Lyon meetup #01**](https://mobilizon.fr/events/b8577678-d072-4d9a-9562-974715f1dfbb)
+* 2022-02-04 | Brussels, BE | [FOSDEM](https://fosdem.org/)
```
Comment (andrewpollack) [inline draft/2023-01-25-this-week-in-rust.md:197]:
```suggestion
* 2023-02-04 | Brussels, BE | [FOSDEM](https://fosdem.org/)
```

**Review by andrewpollack — CHANGES_REQUESTED** [review]
Looks great! One minor change, then good to merge

File: draft/2023-01-25-this-week-in-rust.md (hunk 1 above)
Comment (mariannegoldin) [reply draft/2023-01-25-this-week-in-rust.md]: Thanks for catching this! Would be handy to have that year correct :) Fixing.

**Comment by bennyvasquez** [comment]
Glorious, as always. Thank you!

**Pull request merged**